CHANGELOG.md

```diff
@@ -6,10 +6,62 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
+## [1.2.0-rc.2] - 2024-06-26
+
+> TRUE or FALSE, it's a problem!
+
+### Important Notes
 
  * libcontainer/cgroups users who want to manage cgroup devices need to explicitly
    import libcontainer/cgroups/devices. (#3452, #4248)
+ * If building with Go 1.22.x, make sure to use 1.22.4 or a later version.
+   (see #4233 for more details)
+
+### Added
+
+ * CI: add actuated-arm64. (#4142)
+
+### Fixed
+
+ * Set the default value of CpuBurst to nil instead of 0. (#4210, #4211)
+ * runc exec: fix setting rlimit_nofile. (#4195, #4265, #4282, #4290)
+ * libct/cg/fs: fix setting rt_period vs rt_runtime. (#4094, #4258)
+ * script/*: fix gpg usage wrt keyboxd. (#4189)
+ * Fix a debug log message for user namespaces in nsexec. (#4311)
+ * libct/cg: write unified resources line by line. (#4186)
+ * libct.Start: fix locking, do not allow a second container init. (#4271)
+ * Fix tests in debian testing (mount_sshfs.bats) - minimal fix. (#4245)
+ * Fix checkpoint/restore tests on actuated-arm64. (#4276)
+ * Fix codespell warnings. (#4291)
+ * libct/cg/dev: fix TestSetV1Allow panic. (#4295)
+ * tests/int/scheduler: require smp. (#4298)
+ * tests/int: swap-related fixes. (#4188)
+
+### Changed
+
+ * libcontainer: allow containers to make apps think fips is enabled/disabled for testing. (#4246)
+ * libct: decouple libct/cg/devices. (#4248)
```

Comment:
I'd remove this line, the changelog entry for this PR has alredy been added here: https://github.com/opencontainers/runc/pull/4317/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR15